buildpack.toml

```diff
@@ -11,10 +11,20 @@ api = "0.7"
     uri = "https://github.com/paketo-buildpacks/node-engine/blob/main/LICENSE"
 
 [metadata]
-  include-files = ["bin/build", "bin/detect", "bin/run", "bin/optimize-memory", "bin/inspector", "buildpack.toml"]
-  pre-package = "./scripts/build.sh"
-  [metadata.default-versions]
-    node = "20.*.*"
+  include-files = [
```

Comment:
Can you also include the optimize-memory and inspector? I think the setup-symlinks are not produced.